add the ability to set a subset of feature flags from Info.plist (#891)
apple/swift-docc: Sources/SwiftDocC/SwiftDocC.docc/SwiftDocC/AddingFeatureFlags.md

> # Adding Feature Flags

Reviewer: Love these new docs 👍 thank you!

Develop experimental features by adding feature flags.

## Overview

> When developing a new feature in Swift-DocC, it's recommended to make it optional while it's being
> actively developed by creating a command-line flag that enables the new behavior. This flag can then
> be used to set a flag in the ``FeatureFlags``' ``FeatureFlags/current`` instance, which is then
> available for the rest of the compilation process.

Reviewer: Can you use less passive voice and more active voice throughout?

```suggestion
Make new features in Swift-DocC optional during active development by creating a command-line flag that
enables the new behavior. Then set the new flag in the ``FeatureFlags``' ``FeatureFlags/current`` instance,
making it available for the rest of the compilation process.
```
